tests/components/history/test_init.py

```diff
@@ -39,25 +38,26 @@ def listeners_without_writes(listeners: dict[str, int]) -> dict[str, int]:
 
 
 @pytest.mark.usefixtures("hass_history")
-def test_setup() -> None:
+async def test_setup() -> None:
     """Test setup method of history."""
     # Verification occurs in the fixture
 
 
-def test_get_significant_states(hass_history) -> None:
+async def test_get_significant_states(hass_history, hass: HomeAssistant) -> None:
```

Comment:
```suggestion
async def test_get_significant_states(hass: HomeAssistant, hass_history) -> None:
```

Reply:
This is not possible, the recorder fixture must be set up first